Apps/Immunization/test/unit/Controllers.Test/ImmunizationControllerTests.cs

```diff
@@ -43,47 +42,47 @@ public class ImmunizationControllerTests
+                    ProviderOrClinic = "Mocked Clinic",
+                    Immunization = new ImmunizationDefinition
+                    {
+                        Name = "Mocked Name",
+                        ImmunizationAgents =
+                        [
+                            new()
+                            {
+                                Name = "mocked agent",
+                                Code = "mocked code",
+                                LotNumber = "mocked lot number",
+                                ProductName = "mocked product",
+                            },
+                        ],
+                    },
+                };
+
+            // Blank agent
+            ImmunizationEvent event2 = new()
+            {
+                DateOfImmunization = DateTime.Today,
+                Immunization = new ImmunizationDefinition
+                {
+                    Name = "Mocked Name",
+                    ImmunizationAgents = [],
+                },
+            };
+
             RequestResult<ImmunizationResult> expectedRequestResult = new()
             {
                 ResultStatus = ResultType.Success,
                 TotalResultCount = 2,
                 ResourcePayload = new(
                     new LoadStateModel
                         { RefreshInProgress = false },
-                    new List<ImmunizationEvent>
-                    {
-                        new()
-                        {
-                            DateOfImmunization = DateTime.Today,
-                            ProviderOrClinic = "Mocked Clinic",
-                            Immunization = new ImmunizationDefinition
-                            {
-                                Name = "Mocked Name",
-                                ImmunizationAgents = new List<ImmunizationAgent>
-                                {
-                                    new()
-                                    {
-                                        Name = "mocked agent",
-                                        Code = "mocked code",
-                                        LotNumber = "mocekd lot number",
-                                        ProductName = "mocked product",
-                                    },
-                                },
-                            },
-                        },
-
-                        // Add a blank agent
-                        new()
-                        {
-                            DateOfImmunization = DateTime.Today,
-                            Immunization = new ImmunizationDefinition
-                            {
-                                Name = "Mocked Name",
-                                ImmunizationAgents = new List<ImmunizationAgent>(),
-                            },
-                        },
-                    },
-                    new List<ImmunizationRecommendation>()),
+                    [event1, event2],
+                    []),
```

Comment:
Nit: now that these lines are much shorter, they can probably all fit on a single line.

Reply:
Need to configure Rider.  Change next time.